server/src/Controllers/newsController.ts

```diff
@@ -0,0 +1,30 @@
+import { Request, Response } from 'express';
+import newsLibrary from '../Libraries/newsLibrary';
+
+
+
+export const addNews = async (req: Request, res: Response): Promise<void> => {
+    try {
+        const {user_id , news_content} = req.body;
```

Comment:
we should have type for every body sent in a types folder outside

Reply:
DONE  ✅